movo_desktop/movo_viz/rviz/robot_.rviz

```diff
@@ -634,7 +639,7 @@ Visualization Manager:
       Value: true
     - Class: rviz/Image
       Enabled: false
-      Image Topic: /kinect2/sd/image_color_rect
+      Image Topic: /camera/depth/image_rect_raw
```

Comment:
might change this topic to be the color version but this name seemed closer to the previous